api/src/main/java/org/pmiops/workbench/access/AccessTierService.java

```diff
@@ -0,0 +1,18 @@
+package org.pmiops.workbench.access;
+
+import java.util.List;
+import org.pmiops.workbench.db.model.DbAccessTier;
+import org.pmiops.workbench.db.model.DbUser;
+
+public interface AccessTierService {
+  String REGISTERED_TIER_SHORT_NAME = "registered";
+
+  List<DbAccessTier> getAccessTiersForUser(DbUser user);
+
+  // currently the required tier in some contexts
+  DbAccessTier getRegisteredTier();
```

Comment:
nit: exposing this and the lookup by string feels a bit redundant to me, maybe it's temporary though